### DISCUSSION TITLE
Fix to broker currency v9.0.5.7???

### DISCUSSION THREAD

**Discussion by Wraithlord2015** [Torhal/Broker_Currency#11] [Q&A]
I don't want to spam up the issue tickets with questions etc.. but has anyone managed to fix the problems with version 9.0.5.7?

I've been smashing my head against the wall and can't figure it out.

**Comment by TreizeEU** [comment]
I don't know enough of lua coding than be able to add new currencies to the old 9.0.2.1 version, so I haven't tried to figure out a proper fix for the latest version.

That being said, I don't think we'll know for certain whether this addon has been abandoned unless Torhal either puts up a new version, or speaks up somewhere? To my knowledge, it would be his second abandoned addon after Archy in the end of Legion.
